invokeai/app/invocations/generate.py

```diff
@@ -150,6 +150,9 @@ def invoke(self, context: InvocationContext) -> ImageOutput:
         )
         mask = None
 
+        if self.fit:
```

Comment:
Shouldn't this already be happening inside the below `generate` call?

Reply:
It's not, and I had a separate PR for this fix, which is merged: https://github.com/invoke-ai/InvokeAI/issues/3308

Moot point once we move to latents tho